python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-accounting.j2

```diff
@@ -6,48 +6,50 @@
 {# eos - aaa accounting #}
 {% if aaa_accounting is arista.avd.defined %}
 {%     if aaa_accounting.exec.console.type is arista.avd.defined %}
```

Comment:
I think, the `type` key should be required here

Reply:
It could be breaking change but yes ideally as per j2 and EOS it should be a required key 
cc @gmuloc @ClausHolbechArista

Reply:
Please enforce required keys in the template and not in the schema (for eos_cli_config_gen only!)